ra/saphana-controller-lib

```diff
@@ -69,16 +68,15 @@ function saphana_usage() {
     The 'validate-all' operation reports whether the parameters are valid
     The 'methods'      operation reports on the methods $0 supports
     The 'reload'       operation allows to adapt resource parameters
-
-EOF
+    "
     return "$rc"
 } # end function saphana_usage
 
 function saphana_print_description() {
     # function saphana_print_description - ouput the XML description of the resource agent
     # called by: saphana_meta_data
-    cat <<END_OF_DESCR
-<shortdesc lang="en">Manages two SAP HANA database systems in system replication (SR).</shortdesc>
+    # shellcheck disable=
```

Comment:
the shellcheck line is missing 'SC2016', should be
'# shellcheck disable=SC2016'